### PR TITLE
Add AWS/EKS-Helm Jenkins module

### PR DESCRIPTION
**Pull request by spartan-tainguyen**
## Summary

Add a new module to configure Jenkins on AWS Kubernetes via helm

## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [x] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

## Related Issues

https://www.notion.so/c0x12c/Infra-Rollout-new-Terraform-to-Service-BridgeHealth-15e01fb05bf18092b67af29a49abfc9a

### CODE REVIEW

File: aws/eks-helm/jenkins/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_aws"></a> [aws](#provider\_aws) | n/a |
```
Comment (spartan-toannhan) [inline aws/eks-helm/jenkins/README.md:10]: Please defune the provider

File: aws/eks-helm/jenkins/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<!-- BEGIN_TF_DOCS -->
+## Requirements
```
Comment (spartan-toannhan) [inline aws/eks-helm/jenkins/README.md:2]: Please define the requirement

Comment (spartan-tainguyen) [reply aws/eks-helm/jenkins/README.md:2]: I have updated the PR with what you suggested

File: aws/eks-helm/jenkins/efs-volume.tf [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+resource "kubernetes_persistent_volume" "jenkins_home" {
+  depends_on = [aws_efs_access_point.jenkins_home]
+
+  metadata {
+    name = "jenkins-home-pv"
+  }
+
+  spec {
+    capacity = {
+      storage = "30Gi"
```
Comment (spartan-toannhan) [inline aws/eks-helm/jenkins/efs-volume.tf:32]: Use the variable

**Review by spartan-toannhan — APPROVED** [review]

**Pull request merged**